backends/redis.go

```diff
@@ -0,0 +1,118 @@
+package backends
+
+import (
+	"fmt"
+	"net/url"
+	"strconv"
+
+	"github.com/go-redis/redis"
+)
+
+type RedisBackend struct {
+	client *redis.Client
+}
+
+func NewRedisBackend(connString string) (*RedisBackend, error) {
+	u, _ := url.Parse(connString)
```

Comment:
Is there a standard for `protocol://host:options` strings for Redis? If there is, I couldn't find it. This should probably be documented as it relies on the fairly strange `redis://<host>:<port>?<optional_password>&<optional_db_num>`